cli/Makefile.toml

```diff
@@ -102,8 +102,9 @@ mac_alias = "mac-profile-dev"
 [tasks.fetch-assets]
 description = "Fetch the Pathfinder tar.gz"
 script = '''
+CACHE_BREAKER=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)
```

Comment:
Will that work on MacOS? I don't think they have /dev/urandom

Reply:
Yes https://en.wikipedia.org/wiki//dev/random#macOS,_iOS_and_other_Apple_OSes